src/app/query/bitcoin/balance/btc-taproot-balance.hooks.ts

```diff
@@ -70,3 +77,32 @@ export function useTaprootAccountUtxosQuery() {
     }
   );
 }
+
+export function useCurrentTaprootAccountUninscribedUtxos() {
+  const { data: utxos = [] } = useTaprootAccountUtxosQuery();
+  const query = useTaprootInscriptionsInfiniteQuery();
```

Comment:
Is it possible to use `select` so we can return the whole query?

Reply:
I now realise that `useTaprootInscriptionsInfiniteQuery` is wrong name here. not so long ago I added fetching native segwit as well in this query. @fbwoolf wonder if you can rename it as will in this pr please